.github/workflows/test.yml

```diff
@@ -144,14 +145,53 @@ jobs:
           yarn tailwind-preset build
           yarn components build
       - name: Test Components
-        run: yarn components test-ct --shard ${{ matrix.shard }}
-      - name: Upload Results
-        uses: actions/upload-artifact@v4
+        run: yarn components test-ct --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }} --reporter=blob
+      - name: Upload blob report artifacts
         if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: blob-report-${{ matrix.shardIndex }}
+          path: packages/orbit-components/blob-report
+          retention-days: 2
```

Comment:
If the PR lives more than 2 days, the report will be gone?

Reply:
Then we don't even need 2 days, right? What's the criterion for selecting this number?

Reply:
They actually document for 1 day. I went for 2 days because as I was experimenting thins on this PR I might've wanted to go to old artifacts and recover old reports. But yeah, I'm ok with reducing it to 1 day

Reply:
No strong opinion, just trying to understand the logic. I don't think it will hurt to keep 2 days, as you like.